### ISSUE TITLE
Room settings no topic (design only): needs some work to redesign/style

### ISSUE THREAD

**Issue by bradlegge** [vector-im/riot-web#1755]
Room settings no topic (design only): needs some work to redesign/style the form including additional room settings
- impl DESCOPED FOR NOW, unless there is time once design is ready

**Comment by bradlegge** [comment]
Screen URL:
https://app.zeplin.io/project.html#pid=573bea252e3e73497df13165&sid=5746423293372c320a42b344

**Comment by lampholder** [comment]
Presumably if we are going to draw inspirition from ribot's stuff it'll be through browsing the zeplin directly rather than going via github riot-web issues. I think we can close this.
